webrender/src/platform/unix/font.rs

```diff
@@ -53,7 +53,56 @@ unsafe impl Send for FontContext {}
 
 extern "C" {
     fn FT_GlyphSlot_Embolden(slot: FT_GlyphSlot);
-    fn FT_GlyphSlot_Oblique(slot: FT_GlyphSlot);
+}
+
+const OBLIQUE_SKEW_FACTOR: f32 = 0.2;
```

Comment:
Why is this value different between mac and linux? We could perhaps add a comment referencing where these numbers are based on.

Reply:
Gecko uses different skew factors for all 3 platforms to more closely match the native skewing of the various APIs.